### PR TITLE
Added check to avoid crash when no KPI selected and new version/facto…

### PR DESCRIPTION
**Pull request by jjenthought**
…ry description

### CODE REVIEW

File: enthought_example/version.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = '0.3.0.dev0'
```
Comment (corranwebster) [inline enthought_example/version.py]: Should probably be `0.4.0.dev0` now?

File: enthought_example/example_mco/example_mco.py [hunk 2]
```diff
@@ -96,6 +96,7 @@ def run(self, model):
 
             # When there is new data, this operation informs the system that
             # new data has been received. It must be a dictionary as given.
-            self.notify_new_point([DataValue(value=v) for v in value],
-                                  [DataValue(value=v) for v in out_data],
-                                  [1.0/len(out_data)]*len(out_data))
+            if len(out_data) != 0:
```
Comment (corranwebster) [inline enthought_example/example_mco/example_mco.py]: Possibly want to send the input values if there is no output data?

File: enthought_example/version.py (hunk 1 above)
Comment (jjenthought) [reply enthought_example/version.py]: There hasn't been a 0.3 release for this plugin yet, it was unclear to me whether plugin <-> bdss compatibility should be tracked via `version.py` or if the `PLUGIN_VERSION` (integer!) attribute in `example_plugin.py` should be used instead.
For me `__version__` should be a devlopment thing independent of the bdss versions and `PLUGIN_VERSION` should be renamed and refer to the  min(/max) compatible version of the bdss.

Comment (jjenthought) [reply enthought_example/version.py]: Well I guess you already set the bdss compatibility stuff in setup.py

**Comment by jjenthought** [comment]
@corranwebster Was going through my pull requests, didn't realise this had been A. reviewed and B. left for so long - I actually thought it was already in master. Sorry about that

**Pull request closed without merging**